alfalfa_worker/jobs/openstudio/__init__.py

```diff
@@ -2,5 +2,5 @@
 from pathlib import Path
 
 lib_dir = Path(os.path.dirname(__file__), 'lib')
-from .create_run import CreateRun
-from .step_run import StepRun
+# from .create_run import CreateRun
+# from .step_run import StepRun
```

Comment:
Removed to prevent source files from being automatically loaded. 

The `__init__.py` is run when the module gets imported, and in the dispatcher unit tests this can cause it to crash as `StepRun` has dependencies (`openstudio`, `pyenergyplus`) which may not be present on a developer's computer and are not present in the CI runner. 

This could be avoided by using a job other than `CreateRun` to confirm that the dispatcher is able to properly resolve job paths.